ui.apps/src/main/content/jcr_root/apps/wcm/core/content/editor/_jcr_content/content/items/content.xml

```diff
@@ -0,0 +1,82 @@
+                        granite:class="pageinfo"
+                        granite:id="pageinfo-popover"
+                        granite:title="Page Information"
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/popover">
+                        <items jcr:primaryType="nt:unstructured">
+                            <list
+                                granite:id="pageinfo-data"
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="cq/gui/components/authoring/pageinfo/list">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <copypublishurl
+                                        granite:class="foundation-toggleable-control"
+                                        granite:id="sites-publishurl-button"
+                                        granite:title="Copy Publish URL"
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/button"
+                                        sling:orderBefore="viewinadmin"
+                                        text="Copy Publish URL">
+                                        <granite:data
+                                            jcr:primaryType="nt:unstructured"
+                                            foundation-toggleable-control-target="#aem-sites-show-publish-url"/>
+                                        <granite:rendercondition
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="granite/ui/components/coral/foundation/renderconditions/and">
+                                            <condition1
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/renderconditions/simple"
+                                                expression="${empty paramValues['item'][1] ? 'true' : 'false'}"/>
+                                            <condition2
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="acs-commons/renderconditons/ispublished"
+                                                resourcePath="${requestPathInfo.suffix}"/>
+                                            <condition3
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/foundation/renderconditions/feature"
+                                                feature="com.adobe.acs.commons.wcm.impl.copysitespublishurlfeature.feature.flag"/>
```

Comment:
Cant remember - can we use this to include the clientlib, instead of adding it via the category name.

Reply:
Not sure, I will have to check.

Reply:
I have created a wrapper for the clientlib, so this should be OK.